package.json

```diff
@@ -4,14 +4,14 @@
   "description": "",
   "main": "src/index.ts",
   "engines": {
-    "node": "16.*",
-    "npm": "8.*"
+    "node": "14.*",
+    "npm": "6.*"
```

Comment:
Should be 7.*

Reply:
I believe 7 is not supported in the pipeline so will leave at 6. I will bring this up with the platform team.